testpages/hello_world_banner_non-sf.html

```diff
@@ -53,9 +53,9 @@
       });
     }
 
-    setTimeout(function () {
-      sendAdserverRequest();
-    }, PREBID_TIMEOUT);
+    // setTimeout(function () {
```

Comment:
Please remove the commented code here and other places

Reply:
@jaiminpanchal27 I have removed the commented lines.